### ISSUE TITLE
Import-Excel converting column to scientific notation and corrupting value.

### ISSUE THREAD

**Issue by D1gitalDR** [dfinke/ImportExcel#830]
I am trying to automate the importing of a county file and convert it to csv. The issue I am having is that the account number is getting converted to scientific notation.

```Import-Excel BacktaxRaw_FL_Flagler2.xlsx | Export-Csv ExportTest.csv```

If I tried formatting number fields to be text fields using 

```PS C:\testdata> Import-Excel BacktaxRaw_FL_Flagler.xlsx | Export-Excel ExportTest.xlsx -Show -AutoSize -NumberFormat '@'```

The problem here is that the account number last few digits are replaced with Zero. Example. The original number is ''1914310000010100016", Import is converting it to "1.9143100000101E+18 and finally the export is getting converted to "1914310000010100000". As you can see the 16 at the end is replaced with 00.

Ironically if I import the data into a new excel file and specify importing the data through excel and define the type as text then I can use Import-excel to convert to csv with no issue. This is telling me that Import-Excel is struggling to import the data from the original excel file.  Can someone help me with this issue please?

I have attached the import file.
[BacktaxRaw_FL_Flagler.xlsx](https://github.com/dfinke/ImportExcel/files/4559043/BacktaxRaw_FL_Flagler.xlsx)

**Comment by dfinke** [comment]
Try `Import-Excel BacktaxRaw_FL_Flagler2.xlsx -AsText <PropertyName>`

**Comment by D1gitalDR** [comment]
I tried:
```
Import-Excel BacktaxRaw_FL_Flagler.xlsx -AsText 'System.String[]' | Export-Excel ExportTest.xlsx -Show -AutoSize
```

No luck. it produced "1914310000010100000" still corrupting the original value.

**Comment by dfinke** [comment]
You want to use the property name there, not the data type.

**Comment by D1gitalDR** [comment]
Sorry maybe I am not understanding the syntax needed but I tried all of these options:

```
Import-Excel BacktaxRaw_FL_Flagler.xlsx -AsText '*' | Export-Excel ExportTest.xlsx -Show -AutoSize
Import-Excel BacktaxRaw_FL_Flagler.xlsx -AsText 'A*' | Export-Excel ExportTest.xlsx -Show -AutoSize
Import-Excel BacktaxRaw_FL_Flagler.xlsx -AsText 'Account Number' | Export-CSV ExportTest.csv
Import-Excel BacktaxRaw_FL_Flagler.xlsx -AsText '[Account Number]' | Export-Excel ExportTest.xlsx -Show -AutoSize
```
The column is A the header name is "Account Number". Not sure if the space is causing the problem or if you mean something different by property name. 

I really appreciate the help!

**Comment by dfinke** [comment]
`import-excel .\BacktaxRaw_FL_Flagler.xlsx -AsText 'Account Number' | select -First 3`

```
Account Number      Issued Date (Year) Tax Yr Cert # A: Account Balance Amount Interest Start Date B: Face Amount Interest Rate Date Redeemed C: Redemption Amt
--------------      ------------------ ------ ------ ------------------------- ------------------- -------------- ------------- ------------- -----------------
0712310650000D00020               2013   2012      0                         0             41426.5        2181.78             0                         2747.29
1914310000010100000               2014   2011      0                         0             41791.5        1530.67             0                                
0711310377000A00005               2011   2010      0                         0             40695.5          81.94             0 42842.5                  175.46
```

**Comment by D1gitalDR** [comment]
Unfortunately the second value is corrupt, it should be 1914310000010100016. All numbers that start with 0 are correct. All others are rounded. 

Just for your reference I uploaded a fixed version of the original file. I was able to correct the original file by using Excels import wizard and defining the first column as text. I am hoping to do the same with your awesome script. Any ideas?

[BacktaxRaw_FL_Flagler_fixed.xlsx](https://github.com/dfinke/ImportExcel/files/4560704/BacktaxRaw_FL_Flagler_fixed.xlsx)

**Comment by D1gitalDR** [comment]
After doing some additional research I did a deeper dive to view the raw source code of the above file. I unzipped the contents of the BacktaxRaw_FL_Flagler.xlsx file and navigated to the \xl\worksheets\sheet1.xml file. I opened it in notepad and saw first hand that none of the values I am looking for in column A are stored in scientific notation. They are indeed stored uncorrupted. Which means every attempt I have made to retrieve the data has resulted in data corruption. Even using Excels own data import wizard will alter the values.

[Screen shot](https://www.screencast.com/t/kJ2iHWuIFov) So you don't need to download file

If you want, download the raw xml file here:
[Raw XML file](https://www.lumentumllc.us/sheet1.xml)
